src/Sulu/Component/HttpCache/HttpCache.php

```diff
@@ -93,6 +93,6 @@ private function getReverseProxyTtl(Response $response)
             return $maxAge - $response->getAge();
         }
 
-        return null;
+        return;
```

Comment:
Of those 136 the ones I checked are documented return values for functions which would could return a value. If the value is not intended to be used we should use `return;` I think. /cc @chirimoya